ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -349,6 +349,7 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
                                              target: nil,
```

Comment:
have we tried to create a regular system barbuttonitem and use the backbuttonimage api  to update the image, and see if that would give us the right accessibility hints without doing our own localziation?

Reply:
This is a regular bar button item we are using to replace the native back button. Unfortunately, we need to specify its accessibility label if we want VO to say "back button". I guess we could revert the nav bar to using the native back button in order to avoid the localization but @amgleitman might have more context as to why we're using a custom back button item here.

Reply:
[Here](https://github.com/microsoft/fluentui-apple/commit/028b81bc768005933fabc56083149f0c1a57b446) is this commit that added the custom back button in the first place.

I don't remember the exact context, but looking at this, I think it had something to do with the fact that the system back button gets hidden by the overlay we use for leading-aligned title views. Another added benefit if I recall correctly is that the system back button won't be aligned vertically in the compact-versus-regular case (the whole 32px vs 44px nav bar struggle), so that's another thing to look out for.

Reply:
but in this bug case we aren't overlaying the leading edge title view right? if it is meant to be vanillar system navigation bar why do we need to do it?

Reply:
Right, we aren't overlaying the leading title view but the back button showing up is still the custom one and not the native back button. Looking at the view hierarchy in our demo, it seems the native back button is always hidden on system, leading and landscape.

Reply:
Would that be the ideal fix though - go back to using the native back button?

Reply:
According to Adam the native back button was causing problems on the nav bar. If we can reliably fix those without relying on a custom back button then going back would be the ideal fix. I'll have to investigate more to know for sure.